share/spice/movies/cast/movies_cast.js

```diff
@@ -0,0 +1,70 @@
+(function (env) {
+    'use strict';
+
+    // This is needed to prevent jQuery from adding '_={timestamp}' to the end
+    // of the URL, which breaks the call to getScript()
+    $.ajaxSetup({ cache: true });
+
+    var DETAIL_ENDPOINT = '/js/spice/movies/details/';
+    var POSTER_URL = 'http://image.tmdb.org/t/p/w185';
+    var MOVIE_URL = 'http://www.themoviedb.org/movie/';
+
+    env.ddg_spice_movies_cast = function(data) {
+        // This isn't exactly pretty, but I can't think of any
+        // more graceful way to handle this.
+        var hasFailed = (
+            !data ||
+            !data.results ||
+            !data.results[0] ||
+            !data.results[0].id
+        );
+
+        if (hasFailed) {
+            return Spice.failed('movies_cast');
+        }
+
+        $.getJSON(DETAIL_ENDPOINT + data.results[0].id, function(api_result) {
+            var hasFailed = (
+                !api_result ||
+                !api_result.results ||
+                !api_result.results.length
+            );
+
+            if (hasFailed) {
+                return Spice.failed('movies_cast');
+            }
+
+            Spice.add({
+                id: 'cast',
+                name: 'Movies',
+                data: api_result.results,
+                meta: {
+                    sourceName: 'TheMovieDb',
+                    sourceUrl: 'http://www.themoviedb.org/person/'
+                        + api_result.results[0].id,
```

Comment:
I can't show you the broken link right now (Kwixer is back to showing as the IA on beta right now) BUT the link to Johnny Depp was linked to id `241259` which is the movie ID of Alice Through The Looking Glass, not Johnny's ID, `85`. Hope that helps explain it.